incubator/etcd/README.md

```diff
@@ -38,15 +38,16 @@ The following tables lists the configurable parameters of the etcd chart and the
 | Parameter               | Description                        | Default                                                    |
 | ----------------------- | ---------------------------------- | ---------------------------------------------------------- |
 | `Name`                  | Spark master name                  | `etcd`                                                     |
-| `Image`                 | Container image name               | `gcr.io/google_containers/etcd-amd64`                      |
-| `ImageTag`              | Container image tag                | `2.2.5`                                                    |
+| `Image`                 | Container image name               | `nearform/docker-etcd`                      |
```

Comment:
I have some questions about the image:

* The image is listed as deprecated. Why should we switch to a deprecated image?
* The image doesn't seem to give a high trust factor. It's a custom image without linking to the source that's manually pushed. Why should we trust the image over the Google provided one?
* What's missing from the current container in use?

Reply:
Running as 'root'